proto/build-container/scripts/install-python.sh

```diff
@@ -26,4 +27,5 @@ pip3 install --upgrade pip
 python3 -m pip install --upgrade setuptools
 
 pip3 install --user "pipenv==${PIPENV_VERSION}"
-pip3 install --user "wheel==${WHEEL_VERSION}" "twine==${TWINE_VERSION}"
\ No newline at end of file
+pip3 install --user "wheel==${WHEEL_VERSION}" "twine==${TWINE_VERSION}"
+pip3 install --user "mypy-protobuf==${MYPY_VERSION}"
```

Comment:
Using mypy-protobuf to get .pyi files for free

N.B. Not actually free, we have to do various fixups but still gets us a lot